Submitted by:            Chris Punches <chris.punches@silogroup.org>
Date:                    2021-07-27
Initial Package Version: 10.2.0
Origin:                  Implementation of an FHS convention
Description:             Sets default directory name for 64-bit 
                         libraries to /lib in gcc for pass1

--- gcc/config/i386/t-linux64	2020-07-23 02:35:17.400385168 -0400
+++ gcc/config/i386/t-linux64	2021-07-27 00:25:31.075882858 -0400
@@ -33,6 +33,6 @@
 comma=,
 MULTILIB_OPTIONS    = $(subst $(comma),/,$(TM_MULTILIB_CONFIG))
 MULTILIB_DIRNAMES   = $(patsubst m%, %, $(subst /, ,$(MULTILIB_OPTIONS)))
-MULTILIB_OSDIRNAMES = m64=../lib64$(call if_multiarch,:x86_64-linux-gnu)
+MULTILIB_OSDIRNAMES = m64=../lib$(call if_multiarch,:x86_64-linux-gnu)
 MULTILIB_OSDIRNAMES+= m32=$(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)$(call if_multiarch,:i386-linux-gnu)
 MULTILIB_OSDIRNAMES+= mx32=../libx32$(call if_multiarch,:x86_64-linux-gnux32)
